docs/ONBOARDING.md

```diff
@@ -41,15 +43,23 @@ Once you are part of the twindle team, please join us here to keep in touch.
 - If you are using terminal(Mac) or command prompt(windows), watch this step by step video:
 
 [![tutorial](https://img.youtube.com/vi/7I9StcZt5cI/0.jpg)](https://youtu.be/7I9StcZt5cI).
+<br><br>
+
+[![tutorial](https://img.youtube.com/vi/bPBTumdMhyQ/0.jpg)](https://youtu.be/bPBTumdMhyQ)
+
+<br>
+
 
 4. Do the third PR
+<br>
 
 **What is the third PR about?**
 - Make few changes in the file `TEAM.md` inside the `docs` directory, commit, through the CLI (command line interface) and raise your second Pull Request.  
 
 
 **How to do the third PR?**
 
+- Check this step by step [tutorial](https://github.com/twindle-co/twindle/issues/226)
```

Comment:
Instead of check just do
[Step by Step tutorial](url)

Reply:
ok.

Reply:
corrected.